tools/topology/sof-imx8qxp-cs42888.m4

```diff
@@ -0,0 +1,68 @@
+#
+# Topology for i.MX8QXP board with cs42888 codec
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`esai.m4')
+include(`pcm.m4')
+include(`buffer.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include DSP configuration
+include(`platform/imx/imx8qxp.m4')
+
+#
+# Define the pipelines
+#
+# PCM0 ----> volume -----> ESAI0 (cs42888)
+#
+
+dnl PIPELINE_PCM_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     period, priority, core,
+dnl     pcm_min_rate, pcm_max_rate, pipeline_rate,
+dnl     time_domain, sched_comp)
+
+# Low Latency playback pipeline 1 on PCM 0 using max 2 channels of s24le.
+# Set 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+	1, 0, 2, s24le,
+	1000, 0, 0,
+	8000, 96000, 48000)
```

Comment:
I think this disagrees with the ESAI clock configurations?

Reply:
We use 2 TDM slots, slot width is 32 bit. So 32 x 2 x 48000 = 3072000

So, even if word length is 24bits in order to compute the correct frame clock (because we are running in TDM node) we must use TDM slot width which is 32.

Reply:
I mean, the fact that you stated minimum 8000 and maximum 96000 for the PCM rate.

Reply:
Ok, I see your point now. Will change min/max to 48000 and modify them later.